client/blocks/login/two-factor-authentication/two-factor-actions.jsx

```diff
@@ -77,6 +77,12 @@ class TwoFactorActions extends Component {
 						<a href={ login( { twoFactorAuthType: 'push' } ) }>{ translate( 'The WordPress mobile app' ) }</a>
 					</p>
 				) }
+
+				{ twoFactorAuthType !== 'backup' && (
+					<p>
+						<a href={ login( { twoFactorAuthType: 'backup' } ) }>{ translate( 'A backup code' ) }</a>
```

Comment:
Hi! I've found a possible matching string that has already been translated **24** times:
`translate( 'Invalid Backup Code' )` *ES Score: 10*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/backup-code-login)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).